doc/modules/feature_extraction.rst

```diff
@@ -223,16 +224,17 @@ Since a simple modulo is used to transform the hash function to a column index,
 it is advisable to use a power of two as the ``n_features`` parameter;
 otherwise the features will not be mapped evenly to the columns.
 
+.. topic:: References:
+  * `MurmurHash3 <https://github.com/aappleby/smhasher>`_.
```

Comment:
The reference is not being rendered because it needs an extra space:

```suggestion
.. topic:: References:

  * `MurmurHash3 <https://github.com/aappleby/smhasher>`_.
```

Reply:
I have fixed this, thank you. I have also added the 4 dropdowns that you have mentioned, please review the PR for a merge.